configs/evonet/dapi/nginx/default.conf

```diff
@@ -67,7 +67,7 @@ server {
     }
 
     # Route GRPC-WEB requests to Envoy DAPI API service
-    location /org.dash.platform.dapi.v0.Core {
+    location ~ /org.dash.platform.dapi.v0.Core {
```

Comment:
Why do we need regexp modifier here?

Reply:
We'll get into the `/org.dash.platform.dapi.v[0-9]+.Core` location without it.

Reply:
Should we then escape dots in regexp?

Reply:
it just works

Reply:
Right, due to dot in regexp represents any char. So it will work if a requested service name will be something like `/org1dash2platform3dapi4v56Core`. We shouldn't allow that I guess.